Changelog.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- fossa-deps: Fixed an issue where Rocky linux deps could skip the fossa-deps file during analysis
```

Comment:
Can you add a link to the PR? Also, is that an accurate description of what we're fixing. Based on my reading, it's something like "Fixed an issue where Rockly Linux deps were not supported in the fossa-deps file"

```suggestion
- fossa-deps: Fixed an issue where Rocky Linux deps were not supported in the fossa-deps file ([#1473](https://github.com/fossas/fossa-cli/pull/1473))
```

Reply:
Also, `# Unreleased` should go below `# FOSSA CLI Changelog` and be a level 2 header: `## Unreleased`

Reply:
Thank you! That's a better description!


I also updated the PR template to make this more intuitive. It originally said "`# Unreleased` section at the top," but my markdown senses were tingling.